python/temba_expressions/dates.py

```diff
@@ -203,7 +203,7 @@ def _get_token_possibilities(cls, token, mode):
                         nano = as_int
                     possibilities[Component.NANO] = nano
                 if len(token) == 4:
-                    hour = as_int / 100
+                    hour = as_int // 100
```

Comment:
It's mind boggling that someone decided it was cool to let the meaning of / silently change...